ydb/core/kqp/ut/federated_query/generic/ch_recipe_ut_helpers.cpp

```diff
@@ -6,19 +6,19 @@
 namespace NTestUtils {
 
     TString GetChHost() {
-        return GetEnv("RECIPE_CLICKHOUSE_HOST", "localhost");
+        return "localhost";
     }
 
     ui32 GetChPort() {
-        return FromString<ui32>(GetEnv("RECIPE_CLICKHOUSE_NATIVE_PORT", "1234"));
+        return 19000;
```

Comment:
> What will happen if several tests that use recipe will be started at the same time?

We intentionally disable local parallel runs (avoid `FORK_SUBTESTS` and so on) of tests based on `docker-compose` recipe because they look broken in `ya`. For more details, see link in TG channel :)